edb/ir/typeutils.py

```diff
@@ -87,6 +87,16 @@ def is_cfg_view(
     )
 
 
+def is_excluded_cfg_view(
+    child: s_obj.Object,
+    *,
+    ancestor: s_obj.Object,
+    schema: s_schema.Schema,
+) -> bool:
+    """Used to exclude sys/cfg tables from non sys/cfg views for performance."""
```

Comment:
Include a link to the issue and the comment that we don't really like this anyway